src/js/lib/admin-page/components/ParselySettings.js

```diff
@@ -0,0 +1,272 @@
+						name="disable_amp"
+						value={ settings.disable_amp }
+						label={ __( 'Disable AMP', 'wp-parsely' ) }
+						onChange={ handleInputChange }
+						note={ __( 'Default: On. Disable our AMP tracking if you use a separate system to track AMP content', 'wp-parsely' ) }
+					/>
+					<Setting
+						name="use_top_level_cats"
+						value={ settings.use_top_level_cats }
+						label={ __( 'Use Top-Level Categories', 'wp-parsely' ) }
+						onChange={ handleInputChange }
+						note={ __( 'Default: On. Choose if you want the first top-level category to be mapped to Parse.ly', 'wp-parsely' ) }
+					/>
+					<Setting
+						name="cats_as_tags"
+						value={ settings.cats_as_tags }
+						label={ __( 'Categories as Tags', 'wp-parsely' ) }
+						onChange={ handleInputChange }
+						note={ __( 'Default: On. Choose if you want your non-primary categories to appear as tags', 'wp-parsely' ) }
+					/>
+					<Setting
+						name="track_authenticated_users"
+						value={ settings.track_authenticated_users }
+						label={ __( 'Track Authenticated Users', 'wp-parsely' ) }
+						onChange={ handleInputChange }
+						note={ __( 'Default: On.', 'wp-parsely' ) }
+					/>
+					<Setting
+						name="lowercase_tags"
+						value={ settings.lowercase_tags }
+						label={ __( 'Lowercase Tags', 'wp-parsely' ) }
+						onChange={ handleInputChange }
+						note={ __( 'Default: On. Choose if you want your tags to be converted to lower case', 'wp-parsely' ) }
+					/>
+					<Setting
+						name="force_https_canonicals"
+						value={ settings.force_https_canonicals }
+						label={ __( 'Force HTTPS Canonical URLs', 'wp-parsely' ) }
+						onChange={ handleInputChange }
+						note={ __( 'Default: Off. Choose if you want your canonicals to use the HTTPS scheme', 'wp-parsely' ) }
+					/>
+				</div>
+				<div className={ `tab-body debug ${ displayDiv( 'debug', currentTab ) }` }>
+					{ displayModal ? (
+						<WipeMetadataModal
+							onConfirm={ setMetadataFlag }
+							apikey={ settings.apikey }
+							modalControl={ showModal }
+							setting={ { parsely_wipe_metadata_cache: settings.parsely_wipe_metadata_cache } }
+						/>
+					) : (
+						''
+					) }
+					<Setting
+						name="metadata_secret"
+						value={ settings.metadata_secret }
+						label={ __( 'Metadata Secret', 'wp-parsely' ) }
+						onChange={ handleInputChange }
+						note={ __( 'The metadata secret provided to you by Parse.ly', 'wp-parsely' ) }
+					/>
+					<WipeMetadataCache
+						setting={ { parsely_wipe_metadata_cache: settings.parsely_wipe_metadata_cache } }
+						label={ __( 'Wipe Metadata Cache', 'wp-parsely' ) }
+						onClick={ showModal }
+						note={ __( 'This will wipe all of your site\'s metadata and resend all metadata to Parse.ly', 'wp-parsely' ) }
+					/>
+					<SiteDetails
+						apikey={ settings.apikey }
+						postsToTrack={ settings.track_post_types }
+						pagesToTrack={ settings.track_page_types }
+						pluginVersion={ __('2.5', 'wp-parsely' ) }
```

Comment:
Can this version number be grabbed from somewhere else, rather than having yet another place to remember to update it?

Also, it doesn't need translating.